draft-yasskin-http-origin-signed-responses.md

```diff
@@ -961,6 +966,10 @@ of HTTP/2 frames, or could
    {{?I-D.vkrasnov-h2-compression-dictionaries}} to use any compression state
    from other responses.
 
+To reduce the likelihood of accidental modification by proxies, an
+`application/http2` format should be sure to include a file signature that
+doesn't collide with other known signatures.
```

Comment:
Maybe add a note that responses are encouraged to include `Cache-Control: no-transform`?